package.json

```diff
@@ -347,6 +347,9 @@
       "!node_modules/better-sqlite3/src/*",
       "node_modules/better-sqlite3/build/Release/better_sqlite3.node",
       "!dev-app-update.yml"
+    ],
+    "extraFiles": [
+      "mmdb/GeoLite2-Country.mmdb"
```

Comment:
can you make sure when using a release the paths are resolved correctly too? 
with `yarn build-release` you'll get  a build inside release/

Reply:
I'm not sure I understand.

I've been doing `yarn build-release`, and it's been producing an AppImage that works fine. The Windows binary, too, runs fine when packaged using this path.